test/extended/compatibility.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#
+# Runs extended compatibility tests with a previous version
+source "$(dirname "${BASH_SOURCE}")/../../hack/lib/init.sh"
+source "${OS_ROOT}/test/extended/setup.sh"
+
+# Previous version to test against
+PREVIOUS_VERSION="v1.2.0"
+
+if [[ -n "${PREVIOUS_API:-}" ]]; then
```

Comment:
Ah, sorry, misread the test originally. No need to change the `PREVIOUS_API` flag, but maybe rename to `USE_PREVIOUS_API` to make sure it's obvious that I shouldn't pass in `v1.1.0` in there and expect it to work.